apps/commercetools/package.json

```
@@ -1,36 +1,53 @@
 {
-  "name": "commercetools-sku",
+  "name": "commercetools",
   "version": "1.0.0",
   "private": true,
+  "main": "src/",
   "devDependencies": {
-    "@babel/core": "7.7.2",
-    "@babel/plugin-proposal-class-properties": "7.7.0",
-    "@babel/plugin-transform-runtime": "7.6.2",
-    "@babel/preset-env": "7.7.1",
-    "@babel/preset-react": "7.7.0",
-    "@contentful/contentful-extension-scripts": "^0.15.1",
-    "contentful-cli": "1.2.12",
-    "cssnano": "4.1.10"
+    "@babel/core": "7.4.5",
+    "@babel/plugin-proposal-class-properties": "7.4.4",
+    "@babel/plugin-transform-runtime": "7.4.4",
+    "@babel/preset-env": "7.4.5",
+    "@babel/preset-react": "7.0.0",
+    "@contentful/contentful-extension-scripts": "^0.14.1",
+    "contentful-cli": "^1.1.5",
+    "@testing-library/react": "8.0.4",
+    "@types/array-move": "2.0.0",
+    "@types/jest": "24.0.15",
+    "@types/lodash.get": "^4.4.6",
+    "@types/react": "^16.8.23",
+    "@types/react-dom": "^16.8.5",
+    "@types/react-sortable-hoc": "0.6.5",
+    "@types/webpack-env": "1.13.9",
+    "cssnano": "4.1.10",
+    "typescript": "^3.7.5"
   },
   "dependencies": {
+    "@contentful/forma-36-fcss": "^0.0.34",
+    "@contentful/forma-36-react-components": "^3.30.2",
+    "@contentful/forma-36-tokens": "^0.4.5",
     "@commercetools/api-request-builder": "^5.4.0",
     "@commercetools/sdk-client": "^2.1.1",
     "@commercetools/sdk-middleware-auth": "^6.0.1",
     "@commercetools/sdk-middleware-http": "^6.0.1",
     "@commercetools/sdk-middleware-queue": "^2.1.1",
-    "core-js": "3.4.1",
-    "shared-sku-app": "^1.0.0"
+    "array-move": "2.1.0",
+    "contentful-ui-extensions-sdk": "^3.10.7",
+    "core-js": "3.1.4",
+    "emotion": "^10.0.14",
+    "es6-promise": "^4.2.8",
+    "lodash.get": "^4.4.2",
+    "prop-types": "15.7.2",
+    "react": "16.8.6",
+    "react-dom": "16.8.6",
+    "react-sortable-hoc": "1.9.1"
   },
   "scripts": {
     "start": "contentful-extension-scripts start --serve-only",
     "build": "contentful-extension-scripts build",
     "deploy": "npm run build && aws s3 sync ./build ${STATIC_APPS_BUCKET}/apps/commercetools --acl public-read",
     "test": "contentful-extension-scripts test --env=jsdom --watch",
-    "test:coverage": "contentful-extension-scripts test --env=jsdom --coverage --passWithNoTests",
-    "configure": "contentful space use && contentful space environment use",
-    "login": "contentful login",
-    "logout": "contentful logout",
-    "help": "contentful-extension-scripts help"
+    "test:coverage": "contentful-extension-scripts test --env=jsdom --coverage --passWithNoTests"
```

Should this change to:
```suggestion
    "test:ci": "contentful-extension-scripts test --env=jsdom --coverage --passWithNoTests"
```